src/core/etl/src/Flow/ETL/Row/Entry/UuidEntry.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Flow\ETL\Row\Entry;
+
+use Flow\ETL\Exception\InvalidArgumentException;
+use Flow\ETL\Exception\RuntimeException;
+use Flow\ETL\Row\Entry;
+use Flow\ETL\Row\Reference;
+use Flow\ETL\Row\Schema\Definition;
+
+/**
+ * @implements Entry<\Ramsey\Uuid\UuidInterface|\Symfony\Component\Uid\Uuid, array{name: string, value: string}>
+ */
+final class UuidEntry implements \Stringable, Entry
+{
+    use EntryRef;
+
+    private readonly \Ramsey\Uuid\UuidInterface|\Symfony\Component\Uid\Uuid $value;
+
+    /**
+     * @throws InvalidArgumentException|RuntimeException
+     */
+    public function __construct(private readonly string $name, string|\Ramsey\Uuid\UuidInterface|\Symfony\Component\Uid\Uuid $value)
+    {
+        if ('' === $name) {
+            throw InvalidArgumentException::because('Entry name cannot be empty');
+        }
+
+        if (\is_string($value)) {
+            try {
+                if (\class_exists(\Ramsey\Uuid\UuidInterface::class)) {
```

Comment:
I think this might still fail (but you would need to test it), what I was using in other places was just a plain string `"\Ramsey\Uuid\UuidInterface"`

Reply:
Tested locally and with FCQN it works, tests are skipped properly.